### PR TITLE
Revised the README to target the certification audience.

### PR DESCRIPTION
**Pull request by WilliamDenniss**
Moved WG materials to README-WG.md

### CODE REVIEW

File: README-WG.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+# Kubernetes Conformance
+
+Info regarding the CNCF K8s Conformance Working Group.
```
Comment (dankohn) [inline README-WG.md]: Please kill this line.

File: README-WG.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+The Kubernetes Software Conformance Working Group operates under the aegis of CNCF. We are collaborating with the Kubernetes [Architecture](https://github.com/kubernetes/community/blob/master/sig-architecture/README.md) and [Testing](https://github.com/kubernetes/community/blob/master/sig-testing/README.md) SIGs to help ensure awareness and acceptance of our processes.
```
Comment (dankohn) [inline README-WG.md]: Could you please update this description to describe exactly how we're collaborating with them, as per the blog post.

**Comment by dankohn** [comment]
LGTM after you address my 2 comments. Thanks.

**Comment by WilliamDenniss** [comment]
Updated, PTAL.

**Comment by dankohn** [comment]
+1

**Review by Kato129 — COMMENTED** [review]
Delete this

**Review by timscharfenort8 — CHANGES_REQUESTED** [review]
good work

**Pull request merged**